spec.bs

```diff
@@ -496,6 +522,20 @@ scope=] |debugScope| and an optional [=debug details=] or null
         set |entry|'s [=contribution cache entry/debug details=] to
         |debugDetails|.
 
+To <dfn>determine if a null report should be sent</dfn> given a [=pre-specified
+report parameters=] |preSpecifiedParams|, perform the following steps. They
+return a [=boolean=]:
+1. If |preSpecifiedParams|' [=pre-specified report parameters/context ID=] is
+    not null, return true.
+1. If |preSpecifiedParams|' [=pre-specified report parameters/filtering ID byte
+    size=] is not 1, return true.
```

Comment:
Shall we define an implemention-defined value for the default filtering ID byte size?

Reply:
Even if it’s not implementation defined, maybe we can follow the ARA spec which defines constant values for readability?

Reply:
Yep, added!